src/firefly/js/drawingLayers/FootprintTool.js

```diff
@@ -342,6 +347,23 @@ function getLayerChanges(drawLayer, action) {
             if (!isEmpty(get(drawLayer, ['drawData', 'data']))) {
                 return attachToNewPlot(drawLayer, get(action.payload, ['plotIdAry', '0']));
             }
+            break;
+        case ImagePlotCntlr.CHANGE_CENTER_OF_PROJECTION:
+        case ImagePlotCntlr.ANY_REPLOT:
```

Comment:
you could get a bunch of these events.  Have you check scrolling performance to make sure it is not compromised?